etcdctl/ctlv2/command/util.go

```diff
@@ -78,7 +79,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "http://127.0.0.1:2379,http://127.0.0.1:4001"
```

Comment:
@heyitsanthony also i had done it os specific seperation for onemore reason that is to show localhost for flags used for etcdctlv2 commands when used in windows and display the current behaviour when used for other platforms. For example: for endpoints its mentioned as 127.0.0.1 to be used. while it will not work in windows that way. Similar case is for other command flags of etcdctlv2.